cost/azure/unused_ip_addresses/azure_unused_ip_addresses.pt

```diff
@@ -193,6 +423,10 @@ policy "pol_unused_ip_addresses" do
       field "resourceGroupName" do
         label "Resource Group Name"
       end
+      field "savings" do
+        label "Estimated Monthly Savings"
+        format "currency"
```

Comment:
there isn't a format for currency.  it should be removed as to not confuse the reader.

Reply:
Perhaps there should be a ticket for the Policy Parser to validate that the format is acceptable?

Reply:
removed the currency format